centreon/packages/js-config/cypress/e2e/plugins.ts

```diff
@@ -59,5 +59,18 @@ export default (
     fs.writeFileSync(resultFilePath, JSON.stringify(testRetries, null, 2));
   });
 
+  on("task", {
+    listFilesInDirectory(directoryPath) {
+      return new Promise((resolve, reject) => {
+        fs.readdir(directoryPath, (err, files) => {
+          if (err) {
+            reject(err);
+          } else {
+            resolve(files); // Retourne la liste des fichiers
+          }
+        });
+      });
+    },
+  });
```

Comment:
move it to https://github.com/centreon/centreon/blob/develop/centreon/packages/js-config/cypress/e2e/tasks.ts